apps/weshipit/components/AirtableToggleForm.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from '@weshipit/ui';
+import React, { useState } from 'react';
+
+interface AirtableToggleProps {
+  formLink: string; // should have embed in the URL
+  formHeight: number;
+  buttonText?: string;
+}
+
+export default function AirtableToggleForm({
+  formLink,
+  formHeight,
+  buttonText = 'Start',
+}: AirtableToggleProps) {
+  const [isFormOpen, setIsFormOpen] = useState(false);
+
+  function toggleNavigation() {
+    setIsFormOpen(!isFormOpen);
+  }
+
+  return (
+    <>
+      {isFormOpen ? (
+        <iframe
+          src={formLink}
+          width="100%"
+          height={formHeight}
+          className="m-auto rounded-md"
+          style={{ background: '#ffffff' }}
```

Comment:
can you use tailwind className for `bg-white` and `dark:bg-gray-900` (these are example values replace with the good ones) ?